packages/template-component/package.json

```diff
@@ -15,16 +15,17 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.9.2",
+    "@instructure/emotion": "^7.3.1",
     "@instructure/ui-react-utils": "^7.3.1",
     "@instructure/ui-testable": "^7.3.1",
-    "@instructure/ui-themeable": "^7.3.1",
     "prop-types": "^15"
   },
   "devDependencies": {
     "@instructure/ui-babel-preset": "^7.3.1",
     "@instructure/ui-color-utils": "^7.3.1",
     "@instructure/ui-test-locator": "^7.3.1",
-    "@instructure/ui-test-utils": "^7.3.1"
+    "@instructure/ui-test-utils": "^7.3.1",
+    "@instructure/ui-themes": "^7.3.1"
   },
   "peerDependencies": {
     "react": "^16.8"
```

Comment:
We should also add   "sideEffects": false" here too.

Reply:
done, thanks :)